templates/indicators/add_collecteddata_modal.html

```diff
@@ -1,4 +1,6 @@
 {% block content %}
+{% load i18n %}
```

Comment:
Missing trans tags in this file.

Reply:
This file is not listed in my comment above. I just added the `load` tag to all templates under indicators, but only translated a few of them.